decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Here, you have to set and get the value of instance variables.

ex:
```
attr_accessor :nameable
```

and you do not need to include super() here since we are not implementing anything from the superclass.